PokeAlarm/Manager.py

```diff
@@ -707,14 +707,18 @@ def process_gym(self, gym):
                 "url": gym['url']
             }
 
-        if self.__gym_settings['enabled'] is False:
```

Comment:
If this is about new stickers then I would not expect any changes to the gym code. A new PR for it would be better.